src/constants/networked-trajectories.ts

```diff
@@ -29,8 +29,39 @@ import bloodPlasmaGif from "../assets/card-images/blood_plasma_thumb.gif";
 
 import { TrajectoryDisplayData } from "./interfaces";
 
+const createLink = (url: string, text?: string): string => {
+    return `<a href='${url}'>${text || "here"}</a>`;
+};
+
 const thirdPartyLicensing = (licenseUrl: string) => {
-    return `Please note that the use of this software is subject to third party licensing requirements, which are currently available <a href='${licenseUrl}'>here</a>.`;
+    return `${createLink(licenseUrl, "Third party licensing")} requirements.`;
+};
+
+const softwareUsed = (
+    softwareUrl: string,
+    resources?: {
+        inputsUrl?: string;
+        outputsUrl?: string;
+    }
+): string => {
+    let description = `${createLink(
+        softwareUrl,
+        "Software used"
+    )} to generate data.`;
```

Comment:
```suggestion
    let description = `${createLink(
        softwareUrl,
        "View software used to generate data"
    )}`;
```

Nit: Present link as an action?

Reply:
Will ask UX!